src/Meshes/mesh.ts

```diff
@@ -1289,7 +1289,7 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
                     break;
                 default:
                 case Material.TriangleFillMode:
-                    indexToBind = this._unIndexed ? null : this._geometry.getIndexBuffer();

```

Comment:
well this seems suspicious..Furthermore we are in code lock so I can only merge fixes.
What about mesh that are unindexed. They wont have an index buffer so this code will crash

Reply:
we know that `this._unIndexed` as we test that line 1280 (12 lines above)

Reply:
oh yeah.so true :)